jenkins-integration/beaker/install/shared/stop_sut_background_scripts.rb

```diff
@@ -0,0 +1,12 @@
+require 'json'
+
+step "Stop background scripts on SUT" do
+  pids = JSON.parse(File.read("bg_pids.json"))
+
+  Beaker::Log.notify("Stopping #{pids.count} background scripts")
+  Beaker::Log.notify(JSON.pretty_generate(pids))
+  pids.each do |k, v|
+    Beaker::Log.notify("Stopping script '#{k}' with pid '#{v}'")
+    on(master, "kill #{v}")
```

Comment:
I'll try adding the line.  If it doesn't work immediately I may revert it because I don't want to invest too much time in it for now, but if it does work, I agree it's cleaner.